builtin/mainmenu/tab_help.lua

```diff
@@ -0,0 +1,84 @@
+--Minetest
+--Copyright (C) 2015 shadowzone
+--
+--This program is free software; you can redistribute it and/or modify
+--it under the terms of the GNU Lesser General Public License as published by
+--the Free Software Foundation; either version 2.1 of the License, or
+--(at your option) any later version.
+--
+--This program is distributed in the hope that it will be useful,
+--but WITHOUT ANY WARRANTY; without even the implied warranty of
+--MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--GNU Lesser General Public License for more details.
+--
+--You should have received a copy of the GNU Lesser General Public License along
+--with this program; if not, write to the Free Software Foundation, Inc.,
+--51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+--------------------------------------------------------------------------------
+
+tab_help = {
+	name = "help",
+	caption = fgettext("Help"),
+	cbf_formspec = function (tabview, name, tabdata)
```

Comment:
if you define this function as local function prior defining tab_help you can save a lot of indentions.